lib/tether/tether.go

```diff
@@ -133,6 +136,15 @@ func (t *tether) setup() error {
 		}
 	}
 
+	// Create PID file for tether
+	paths := strings.Split(os.Args[0], "/")
+	err = ioutil.WriteFile(fmt.Sprintf("%s/%s.pid", PIDFileDir, paths[len(paths)-1]),
```

Comment:
There is no need to create a pid file for the tether. You should only create pid files for the processes the tether launches.